transcom/mymove
# [MB-14483] displaying shipment labels for destination SIT service items

## [Jira ticket](https://dp3.atlassian.net/browse/MB-14483) for this change

## Summary

This PR addresses both [MB-14483](https://dp3.atlassian.net/browse/MB-14483) and [MB-13446](https://dp3.atlassian.net/browse/MB-13446)

## Setup to Run Your Code

<details>
<summary>💻 You will need to use two separate terminals to test this locally.</summary>

##### Terminal 1

Start the UI locally.

```sh
make client_run
```

##### Terminal 2

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps



1. Log in as prime simulator
2. Choose move you want to test (I used `RDY4PY`)
3. Request a new service item, Destination SIT
4. Switch to TOO role
5. Find move
6. Navigate to the move history tab
7.  Both shipment label and labeled fields should display properly

## Verification Steps for Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.



### Frontend


## Screenshots
Before:
![image](https://user-images.githubusercontent.com/111928238/202825491-fd8dfcaa-50e2-4da6-a8dc-39fcc996d09f.png)

After 🎉 :
![image](https://user-images.githubusercontent.com/111928238/202825544-5e97b0c2-a62a-4fa6-9737-a20166e5095e.png)
